packages/frontend/src/MobileRoutes.tsx

```diff
@@ -250,15 +249,15 @@ const MobileRoutes: FC = () => {
                                         </TrackPage>
                                     </Route>
 
-                                    <Redirect to="/projects" />
+                                    <Navigate to="/projects" />
```

Comment:
tested going to Metrics Catalog in mobile mode, and go redirected back to /projects